watertap/data/techno_economic/anaerobic_digestion_reactive.yaml

```diff
@@ -0,0 +1,72 @@
+default:
+  biogas_tss_ratio:
+    value: 0.4
+    units: m^3/kg
+  energy_electric_flow_vol_inlet:
+    value: 0.2  # Electricity usage data is unavailable; this value is arbitrary
+    units: kWh/m^3
+  recovery_frac_mass_H2O:
+    value: 1
+    units: dimensionless
+  default_removal_frac_mass_comp:
+    value: 1
+    units: dimensionless
+  removal_frac_mass_comp:
+    tss:
+      value: 1
+      units: dimensionless
+    cod:
+      value: 1
+      units: dimensionless
+    tkn:
+      value: 1
+      units: dimensionless
+    ammonium_as_nitrogen:
+      value: 0
+      units: dimensionless
+    acetic_acid:
+      value: 0
+      units: dimensionless
+  # Notes on reactions: The data from the AMO 1690 team tells us that the
+  # minimum target ratio for g ammonia produced / g TSS in influent is 0.13.
+  # However, the conversion of TSS, COD, and TKN is unknown. So, we have assumed
+  # 50% conversion for TSS, COD, and TKN (an arbitrary choice), and used
+  # 2 * 0.13 = 0.26 for the stoichiometric conversion ratio of NH4.
+  reactions:
+    tss_reaction:
+      key_reactant: tss
+      conversion: 0.5
+      stoichiometry:
+        tss:
+          conversion_ratio: -1
+        ammonium_as_nitrogen:
+          conversion_ratio: 0.26
+        acetic_acid:
+          conversion_ratio: 0.1
+    cod_reaction:
+      key_reactant: cod
+      conversion: 0.5
+      stoichiometry:
+        cod:
+          conversion_ratio: -1
+    tkn_reaction:
+      key_reactant: tkn
+      conversion: 0.5
+      stoichiometry:
+        tkn:
+          conversion_ratio: -1
```

Comment:
The good thing about this is that we can easily update these values/reactions as needed in the future, if given feedback from the team on more appropriate values. So we won't worry about how accurate these assumptions may or may not be for now (e.g., TKN includes ammoniacal nitrogen).

I think your comment covers this message well enough. =)